// Call the main function and catch if there is any error
